packages/editor-ui/src/constants.ts

```diff
@@ -703,3 +703,37 @@ export const TIME = {
 };
 
 export const SUGGESTED_TEMPLATES_FLAG = 'SHOW_N8N_SUGGESTED_TEMPLATES';
+
+/**
+ * Mouse button codes
+ */
+
+/**
+ * Mapping for the MouseEvent.button property that indicates which button was pressed
+ * on the mouse to trigger the event.
+ *
+ * @docs https://www.w3.org/TR/uievents/#dom-mouseevent-button
+ */
+export const MOUSE_EVENT_BUTTON = {
+	PRIMARY: 0,
+	MIDDLE: 1,
+	SECONDARY: 2,
+	BROWSER_BACK: 3,
+	BROWSER_FORWARD: 4,
+};
```

Comment:
Could also use the [`as const`](https://bobbyhadz.com/blog/typescript-as-const) declaration

```suggestion
} as const;
```